src/ventus/wm/windowmanager.js

```diff
@@ -14,9 +14,14 @@ define([
 function($, Window, View, DefaultMode, ExposeMode, FullscreenMode) {
 	'use strict';
 
-	var WindowManager = function () {
+	var WindowManager = function (wmOptions) {
```

Comment:
nit: remove wm- prefix. In the context of the WindowManager "class" this is redundant.